test/parallel/test-webstreams-pipeline.js

```diff
@@ -0,0 +1,412 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const { Readable, Writable, Transform, pipeline } = require('stream');
+const { pipeline: pipelinePromise } = require('stream/promises');
+const { ReadableStream, WritableStream, TransformStream } = require('stream/web');
+const http = require('http');
+
+{
+  const values = [];
+  let c;
+  const rs = new ReadableStream({
+    start(controller) {
+      c = controller;
+    }
+  });
+  const ws = new WritableStream({
+    write(chunk) {
+      values.push(chunk?.toString());
```

Comment:
Nit: since the test is only pushing strings through, perhaps just simply `values.push(chunk)` ?

Reply:
sure updating

Reply:
Fixed